controllers/user.js

```diff
@@ -38,8 +45,8 @@ const getUsers = async (req, res) => {
     const users = await prisma.user.findMany({
       // Request only the name and money fields when querying the database. Makes sure it won't provide anything confidential or dangerous.
       "select": {
+        "id": true,
         "name": true,
-        "money": true,
```

Comment:
You have removed the select statement from all other user data returns, 
but you seem to have forgotten this one.
I have tested the code with deleting the password in a foreach loop and it works as intended